__tests__/integration.spec.js

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2017 American Express Travel Related Services Company, Inc.
```

Comment:
Can you leave the copyright as is?

Reply:
It's probably worthwhile to open a ticket to update all of the American Express copyright notices where they're out of date. For instance, it's highly probable that you've made changes since 2017, and there should have extra notices (or expanded notices e.g. 2017-2020 or 2017, 2018, 2019... Etc, but I didn't think it was my place to make such changes for you guys.

In this case, I just added the ones that I should have added with my ssim work, and that were also appropriate with these changes.

Side note: even if the works you make for hire are owned by American Express, it's not out of bounds for you to annotate that you were a contributor by virtue of your name and email address. Something like, 

Copyright (c) 2019 American Express Travel Related Services Company, Inc. Work performed by Andres Escobar <Andres.Escobar@aexp.com> as a work made for hire.

The reason being that you should be able to advertise the work that you've done even if you're not the owner. In my case, I was updating the notice in accordance with the contributor licence agreement, that says each contributor is an owner of their own derivative works.

If you'd like, I can take a stab at fixing the American Express notices too in a separate pull request. I'd use the git history to find the names dates and contributors.

Reply:
@tklever this is fine right?

Reply:
@tklever can you let us know?

Reply:
Not sure what the hold up for this is on our legal team that needs review since there is a change to the license header... sorry about the delay.

cc @lukasoppermann

Reply:
@omnisip I really appreciate your work on this PR and understand where you are coming from but our legal team is not comfortable with this. Can you revert the license?

Reply:
__tests__/integration.spec.js